### PR TITLE
Rename the Domain Manager role to "manager"

### PR DESCRIPTION
**Pull request by markus-hentsch**
This implements our decision to rename the role[^1] to follow the request by upstream[^2].

[^1]: https://github.com/SovereignCloudStack/minutes/blob/main/iaas/20240313.md#domain-manager-rolepersona-markus-hentsch

[^2]: https://review.opendev.org/c/openstack/keystone-specs/+/903172/2/specs/keystone/2023.1/domain-manager-role.rst#20

### CODE REVIEW

**Comment by markus-hentsch** [comment]
Need to test the changed test script implementation but need a suitable test Keystone instance first.

Marking as draft for now.

**Comment by markus-hentsch** [comment]
> Need to test the changed test script implementation but need a suitable test Keystone instance first.

I've now successfully verified that it works the same using the `manager` role and the tests still pass.
 Verified using a minimal DevStack I built up as documented here: https://gist.github.com/markus-hentsch/b0f7315a9ad6f5dc8621727056df275e

Before removing the draft state I need to also create a corresponding PR for the docs at https://docs.scs.community/docs/iam/domain-manager-setup-and-usage/

**Comment by markus-hentsch** [comment]
> Before removing the draft state I need to also create a corresponding PR for the docs at https://docs.scs.community/docs/iam/domain-manager-setup-and-usage/

Done: https://github.com/SovereignCloudStack/docs/pull/163

File: Standards/scs-0302-v1-domain-manager-role.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ OpenStack Keystone allows for new roles to be created via its API by administrat
 Additionally, each OpenStack API's RBAC can be adjusted through an API policy file (`policy.yaml`) through olso-policy[^4], Keystone included.
```
Comment (josephineSei) [inline Standards/scs-0302-v1-domain-manager-role.md:89]: The headline should be changed to persona, or something alike (a "role model")
#### Introducing a new role and API policy changes

**Review by josephineSei — COMMENTED** [review]
I found a few places, that are still mentioning the role.
It would be good to add a definition of role and persona in the beginning or the glossary.

File: Standards/scs-0302-v1-domain-manager-role.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ track: IAM
 
 SCS Clouds should provide a way to grant Domain Manager rights to SCS Customers which provides IAM self-service capabilities within an OpenStack domain.
 This is not properly implemented in the default OpenStack configuration and requires specific adjustments to the Keystone identity management configuration.
-To avoid conflict with the unscoped `admin` role in OpenStack we want to refer to this new role as "Domain Manager" (`domain-manager`).
+To avoid conflict with the unscoped `admin` role in OpenStack we want to refer to this new persona as "Domain Manager", introducing the `manager` role in the API for domains.
```
[the remark this answers is not in the document]
Comment (markus-hentsch) [reply Standards/scs-0302-v1-domain-manager-role.md:12]: Good point! This is definitely relevant for the glossary.

File: Standards/scs-0302-v1-domain-manager-role.md
[the remark this answers is not in the document]
Comment (markus-hentsch) [reply Standards/scs-0302-v1-domain-manager-role.md:1]: Addressed in cc6c227fe478040fd622d49461b64a7672fb5e68

**Pull request merged**